### DISCUSSION TITLE
Head code?

### DISCUSSION THREAD

**Discussion by erictuvesson** [noodlapp/noodl-docs#20] [General]
Right now there is no dedicated place that explain what head code is used for, what it does, and where to find it.

I am not sure where we should summarize something like this, is there any good place where we could collect this? 

Maybe a page about Project settings? Where we also can go deeper on other things in there.

**Comment by mrtamagotchi** [comment]
I think this sounds like a great idea. There are some neat things in the Project Settings that could need some extra highlighting!

Another idea would be to create a guide on how to do third party integrations, and using the Head Code to load libs from CDN. But that is maybe more "ONE usecase of Head Code" and not "Head Code" in general.
